### ISSUE TITLE
Replace custom administration by flask-admin

### ISSUE THREAD

**Issue by opatut** [opatut/dudel#92]

**Comment by Alexander-Fischer1** [comment]
Is this refering to a backend admin interface? I installed phpmyadmin on the server and ran into some trouble deleting users and polls.

**Comment by opatut** [comment]
Yes, it is. I'm talking about /admin/, where one can view all the polls and users, but sadly not edit them. And it would too much hassle to write that if there is already something like flask-admin for the job.

**Comment by Alexander-Fischer1** [comment]
I looked up an example of flask-admin and it seemed very interesting. I’m looking forward for your future improvements of dudel.
